modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -2,29 +2,29 @@
 = Single Sign-On (SSO)
 :description: SSO allows you to log in to the Aura Console using their company IdP credentials.
 
-label:AuraDB-Virtual-Dedicated-Cloud[]
-label:AuraDS-Enterprise[]
-label:AuraDB-Business-Critical[]
+* *AuraDB Virtual Dedicated Cloud and AuraDS Enterprise* Supports both Organization SSO and Instance SSO which are configurable in the Aura console. 
+Organization owners and organization admins can configure one or more Single Sign-On (SSO) login methods for user authentication.
+* *AuraDB Business Critical* Individual instance level SSO is available by request through support.
```

Comment:
I don't think this is right, it's available to BC instances by default. Will confirm this.

Reply:
Right, but support can set up a special configuration so that only 1 instance in a project with multiple instances has SSO applied to it, and this is only available to Business Critical according to our support colleague. Would that be corret?